src/applications/personalization/profile/components/military-information/MilitaryInformation.jsx

```diff
@@ -74,6 +74,32 @@ const NotInDEERSAlert = () => {
   );
 };
 
+// Request DD214
+const RequestMilServiceRecordsAlert = () => {
+  return (
+    <>
+      <va-alert
+        background-only
+        class="vads-u-margin-y--0"
+        close-btn-aria-label="Close notification"
+        disable-analytics="false"
+        status="info"
+        full-width="false"
+        visible="true"
+      >
+        <p className="vads-u-margin-y--0">
+          <h3 className="vads-u-margin-top--0" slot="headline">
+            Request your military records (DD214)
+          </h3>
+          <a href="https://www.va.gov/records/get-military-service-records">
```

Comment:
I'm not really sure, but I think in general I would lean towards using a relative url here `/records/get-military-service-records`

Reply:
I agree, I'll put that in